<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocatoriPreferiti</class>
 <widget class="QWidget" name="LocatoriPreferiti">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aggiungi - modifica - cerca locatori preferiti</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Cerca per LAT LON</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
        <item>
         <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,1">
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Lon:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="cercaLon">
            <property name="minimum">
             <double>-180.000000000000000</double>
            </property>
            <property name="maximum">
             <double>180.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Lat:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="cercaLat">
            <property name="minimum">
             <double>-90.000000000000000</double>
            </property>
            <property name="maximum">
             <double>90.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="cercaLatLonOK">
          <property name="text">
           <string>Cerca</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Locatori preferiti salvati</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="0">
         <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,1">
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Nome:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Locatore:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="aggiungiLocatore"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="nomeLocatore"/>
          </item>
         </layout>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="aggiungiOK">
          <property name="text">
           <string>Aggiungi
modifica</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QPushButton" name="eliminaOK">
          <property name="text">
           <string>Elimina
selezionato</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QTableWidget" name="listaLocatori">
          <property name="editTriggers">
           <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cercaLat</tabstop>
  <tabstop>cercaLon</tabstop>
  <tabstop>cercaLatLonOK</tabstop>
  <tabstop>aggiungiLocatore</tabstop>
  <tabstop>nomeLocatore</tabstop>
  <tabstop>aggiungiOK</tabstop>
  <tabstop>eliminaOK</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
